### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by oguzhannysr** [Kalit31/Land-Cover-Analysis#31]
@Kalit31 ,The drive link of the pretrained model is not available, can you help?

**Comment by Kalit31** [comment]
Hi @oguzhannysr,
The dataset/pre-trained models were hosted on my university's google drive. The storage had been reduced sometime back, as a result, the link is not available now. Let me check if I can upload the pre-trained models somewhere else and share with you.

**Comment by oguzhannysr** [comment]
@Kalit31 , Hello, I'm waiting for your news. Any news?

**Comment by Kalit31** [comment]
Hi,
Please find the pre-trained models here: https://drive.google.com/drive/folders/1fdoa994tug6UMyH7HpNl7hU5VWT8l-Ej
One is for **Sentinel-2-MSI-1C** images, the other one is for **Sentinel-1-SAR** images.

**Comment by oguzhannysr** [comment]
Can I use the pretrained model in different countries, will the accuracy decrease much?

**Comment by oguzhannysr** [comment]
Also thank you for your efforts. Another question I want to ask is, should the input images to the model be 128*128*4 (rgb, nir)?
I would also like to do training for a different country that I will specify, but do I need to collect ground data with GEE's CART Classifier?
